src/components/backend-ai-import-view.ts

```diff
@@ -120,23 +120,28 @@ export default class BackendAIImport extends BackendAIPage {
           --mdc-select-dropdown-icon-color: var(--general-textfield-selected-color);
           --mdc-select-hover-line-color: var(--general-textfield-selected-color);
           --mdc-list-vertical-padding: 5px;
+          /* Need to be set when fixedMenuPosition attribute is enabled */
+          --mdc-menu-max-width: 300px;
+          --mdc-menu-min-width: 300px;
         }
-        mwc-select.fixed-position > mwc-list-item {
+        mwc-select.fixed-position {
           width: 200px; // default width
+          --mdc-menu-max-width: 200px;
+          --mdc-menu-min-width: 200px;
         }
         mwc-select.github-select {
           margin: inherit;
-          width: 68%;
+          width: 440px;
           margin-bottom: 10px;
           --mdc-theme-primary: var(--general-textfield-selected-color);
           --mdc-select-fill-color: transparent;
           --mdc-select-label-ink-color: rgba(0, 0, 0, 0.75);
           --mdc-select-dropdown-icon-color: var(--general-textfield-selected-color);
           --mdc-select-hover-line-color: var(--general-textfield-selected-color);
           --mdc-list-vertical-padding: 5px;
```

Comment:
I think we can remove `fixedMenuPosition` attribute entirely since it works fine without it. Let's get rid of style definition accordingly.